libindy-crypto/src/cl/issuer.rs

```diff
@@ -388,13 +388,13 @@ impl Issuer {
 
     fn _sign_primary_claim(p_pub_key: &IssuerPublicKey,
                            p_priv_key: &IssuerPrivateKey,
-                           m_2: &BigNumber,
+                           claim_context: &BigNumber,
```

Comment:
Why is it `context` on one place and `claim_context` in other?

Reply:
The problem is we have a `BigNumber` context, i will rename all to `claim_context`